account_credit_control/migrations/9.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openupgradelib import openupgrade
+
+
+def migrate_followup_data(env):
+    env.cr.execute(
```

Comment:
These scripts are not going to work unless you include in OpenUpgrade a renaming from `account_followup` to `account_credit_control`.